tests/unit/api/test_instances.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 The OpenSDS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def test_post_instance(client):
+    # response = client.post('/v1/orchestration/instances')
```

Comment:
remove unused code or add unit test.

Reply:
Plan to add unit test. Have kept it commented as of now